src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -7963,6 +7963,77 @@ void MacroAssembler::cache_wbsync(bool is_pre)
     sfence();
   }
 }
+
+void MacroAssembler::copy64_masked_avx(Register dst, Register src, XMMRegister xmm,
+                                       KRegister mask, Register length, Register temp,
+                                       BasicType type, int offset, bool use64byteVector) {
+  assert(MaxVectorSize >= 32, "vector length < 32");
+  use64byteVector |= MaxVectorSize > 32 && AVX3Threshold == 0;
```

Comment:
When do you expect AVX3Threshold to be 0?

Reply:
As of now when user explicitly pass -XX:AVX3Threshold=0 , default value of AVX3Threshold is 4096.